test/commands/diff.test.ts

```diff
@@ -20,6 +20,12 @@ DevCycle Variable Changes:
 	   Location: test/utils/diff/sampleDiff.js:L10
   4. multi-line-comment
 	   Location: test/utils/diff/sampleDiff.js:L20
+  5. user-object
```

Comment:
perhaps we should have this use a separate diff then the more detailed unit testing of the diffing logic? These tests are mainly meant to test the e2e, as long as a valid diff produces a valid comment I don't think we need to test every one of the matching cases in here

Reply:
True, I'll update these to use an e2e diff rather than the node diff